.scripts/formatJSON5docs.js

```diff
@@ -5,24 +5,29 @@ const json5 = require('json5');
 const cmd = process.argv.join(' ');
 const validate = cmd.includes('--validate');
 
-const json5files = glob.sync('./**/*.json5');
-let counter = 0;
+const json5files = glob.sync('./**/*.json5', {
+  ignore: ['./node_modules/**/*.*'],
+});
+const nonFormattedFiles = [];
 json5files.forEach(fileName => {
   const content = fs.readFileSync(fileName).toString();
   const data = json5.parse(content);
   const newContent = json5.stringify(data, null, 2);
-  if (newContent !== content) {
+  if (newContent.trim() !== content.trim()) {
     if (!validate) {
       fs.writeFileSync(fileName, newContent);
     }
-    counter++;
+    nonFormattedFiles.push(fileName);
   }
 });
 
 console.log(
-  `${counter} json5-file(s) ${validate ? 'should be' : ''} reformatted`,
+  `${nonFormattedFiles.length} json5-file(s) ${
+    validate ? 'should be' : ''
+  } reformatted`,
 );
 
-if (validate && counter) {
+if (validate && nonFormattedFiles.length) {
```

Comment:
Small nitpick:
I think it will be better to use `isValidate` variable name.

Reply:
You missed it here https://github.com/memory-cards/cards/pull/3 ? ))

Reply:
I thought I'd get used to it variable name, but it continues to make me confused.